webpack.config.js

```diff
@@ -24,15 +25,18 @@ module.exports = {
         path: path.resolve(__dirname, 'build'),
         filename: '[name].js'
     },
+    resolve: {
+        symlinks: false
+    },
```

Comment:
This affects the way webpack looks for things such as the `transform-object-rest-spread` plugin (see below) when dealing with `npm link`ed dependencies (say, `scratch-vm` for example). If `symlinks` is true (default) then webpack will look for `scratch-vm/node_modules/babel-plugin-transform-object-rest-spread` only and will fail if it doesn't exist. If `symlinks` is false it will find the one under `scratch-gui/node_modules/...`
See https://github.com/webpack/webpack/issues/985 for multiple pieces of conflicting advice. This setting works for me but may need further tinkering to make sure everything is correct.